packages/core/src/license-finder/parse-license-logical-expression.ts

```
@@ -7,7 +7,9 @@ export function parseLicenseLogicalExpression(
     try {
       return parse(licenseExpression);
     } catch (error) {
-      console.error(error);
+      if (!(error instanceof Error && error.message.startsWith("Unexpected `"))) {
```

Why we filter this error here? 
Why we use console.error here? Can we simply throw this error and use error boundary to handle it?

---

> Why we filter this error here?
If the license isn't parsed by `spdx-expression-parse`, it's still going to be marked as `not found`. The only thing that changes is that we silence an extra error, e.g.: `Unexpected C at offset 0`. The user not only doesn't need this error message, but event if (s)he gets it, it's too cryptic to be of any use.

> Why we use console.error here? Can we simply throw this error and use error boundary to handle it?
If I understand correctly your question, I did try to use the error boundary it before submitting this PR. However, then the whole process crashed. We don't really want that, since (going to my previous point), the license will still be correctly marked as `not found`. So, the point was just to silence this one particular error.

Hope that makes sense, but kindly let me know if you think I'm missing something here :)

---

Ok, I would consider hiding the error behind the verbose flag and maybe treat the all errors from `spdx-expression-parse` in same manner, since user can't do much about it